banks-interface/src/lib.rs

```diff
@@ -10,11 +10,19 @@ use solana_sdk::{
     transaction::{self, Transaction, TransactionError},
 };
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub enum TransactionConfirmationStatus {
+    Unconfirmed,
```

Comment:
More nits: would "unknown" make more sense?   Or by this point do we know for certain that the transaction does in fact exist?
```suggestion
    Unknown,
```

Reply:
Yes, we do know it exists by this point, as the signature is found in the recent bank.

Reply:
Oh wait, so this could be `Recent` then?

Reply:
Sure!
Recent, Max, Optimistic... I guess this sets us up nicely for https://github.com/solana-labs/solana/issues/14414

Reply:
What do we think clients will use `Recent` for? Is it more or less an indication that the transaction was processed?

Reply:
iirc, Serum uses Recent for most queries actually for a snappy UI (at the risk of a later correction to the UI)

Reply:
Yes, it would indicates that the transaction landed.

Reply:
Could we name it "Landed" or "Processed" then? Recent doesn't mean much to a developer I think

Reply:
@jstarry would you feel any better about Recent/Max if this field were renamed to `commitmentStatus`?

Reply:
Nope, I just don't think Recent/Max are dev friendly and it seems like the right time to change them